node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/AllocationSimulator.java

```diff
@@ -11,8 +11,9 @@
 import com.yahoo.vespa.hosted.provision.NodeList;
 import com.yahoo.vespa.hosted.provision.node.Allocation;
 import com.yahoo.vespa.hosted.provision.node.Generation;
+import com.yahoo.vespa.hosted.provision.node.IP;
 
-import javax.swing.JFrame;
+import javax.swing.*;
```

Comment:
Disable wildcard imports for `javax.swing.*`

Reply:
huh, that was a weird exception in IntelliJ - removed and fixed.